app/services/listing_service/store/shape.rb

```diff
@@ -28,6 +29,8 @@ module ListingService::Store::Shape
     [:translations, :array, :optional], # TODO Only temporary
     [:units, :array, :mandatory],
     [:shipping_enabled, :bool, :mandatory],
+    [:url, :string, :mandatory],
```

Comment:
Looks like it's stored as name so it would make sense to call it name throughout the chain.